src/primitive-field/primitive-field.component.ts

```diff
@@ -103,4 +103,15 @@ export class PrimitiveFieldComponent extends AbstractFieldComponent implements O
     return this.schema['disabled'] ? -1 : 1;
   }
 
+  get tooltipPosition() {
+    let tooltipPlacement = 'top';
+    let firstElem = document.getElementById('first');
+    if (firstElem) {
+      // The first top-level element will have the same bottom placement for all it's elements.
+      if (firstElem.querySelector(`[id='${this.pathString}']`)) {
```

Comment:
I am not sure about this repetition, for each primitive component `document.getElementById('first')` will always match and the second query needs to be done.

I am wondering if it is better from `json-editor.component.html` to pass an `Input()` in `any-type-field` with `isLastElement`